---
search:
  boost: 0.5
---
# ebGSEA

Gene Set Enrichment Analysis is one of the most common tasks in the analysis of omic data,  and is critical for biological interpretation. In the context of Epigenome Wide Association Studies (EWAS),  which typically rank individual cytosines according to the level of differential methylation, enrichment  analysis of biological pathways is challenging due to differences in CpG/probe density between genes.

*homepage*: <https://github.com/aet21/ebGSEA>

version | versionsuffix | toolchain
--------|---------------|----------
``0.1.0`` | ``-R-4.2.1`` | ``foss/2022a``

[../0/index.md](0) - [../a/index.md](a) - [../b/index.md](b) - [../c/index.md](c) - [../d/index.md](d) - [../e/index.md](e) - [../f/index.md](f) - [../g/index.md](g) - [../h/index.md](h) - [../i/index.md](i) - [../j/index.md](j) - [../k/index.md](k) - [../l/index.md](l) - [../m/index.md](m) - [../n/index.md](n) - [../o/index.md](o) - [../p/index.md](p) - [../q/index.md](q) - [../r/index.md](r) - [../s/index.md](s) - [../t/index.md](t) - [../u/index.md](u) - [../v/index.md](v) - [../w/index.md](w) - [../x/index.md](x) - [../y/index.md](y) - [../z/index.md](z)

